create_jenkins_job.py

```diff
@@ -456,8 +454,6 @@ def create_job(os_name, job_name, template_file, additional_dict):
                                      '--packages-up-to ' + ' '.join(quality_level_pkgs + testing_pkgs_for_quality_level),
             })
 
-        # configure nightly coverage job on x86 Linux only
-        if os_name == 'linux':
```

Comment:
The description doesn't really give any indication of why this part if the change is made. Could you elaborate a little?

Reply:
That's true, good point.  This is a cleanup; while I was in here, I noticed that this condition is repeated twice with no difference between them (that is, this condition is already above).  Because of that, we can just nest this change inside the previous if statement.

It's just a cleanup, so if you think it is a problem I can pull it out of here.

Reply:
I'd be fine with keeping the change in the PR if it is split into a separate commit and rebase-merged rather than squash merged. Although at that point making two separate PRs isn't that much more effort.

Reply:
I'm just going to drop the change for now, it is not needed for the fix.

Reply:
All right, removed now, ready for another review.